restapi.go

```diff
@@ -710,6 +710,15 @@ func (s *Session) GuildMemberEdit(guildID, userID string, roles []string) (err e
 	return
 }
 
+// GuildMemberEditComplex edits the roles & nickname of a member.
+// guildID  : The ID of a Guild.
+// userID   : The ID of a User.
+// data     : Struct of GuildMemberEditData
+func (s *Session) GuildMemberEditComplex(guildID, userID string, data *GuildMemberEditData) (err error) {
```

Comment:
Modify endpoints always return the entity with modification. It seems you forgot to unmarshal & add it to return parameters.

Additionally I think it's unnecessary to make `data` a pointer.